Kitodo/src/main/java/org/kitodo/production/forms/createprocess/CatalogImportDialog.java

```diff
@@ -360,4 +397,59 @@ public void setAdditionalImport(boolean additionalImport) {
         this.additionalImport = additionalImport;
     }
 
+
+    /**
+     * Get searchTerm.
+     *
+     * @return value of searchTerm
+     */
+    public String getSearchTerm() {
+        return this.searchTerm;
+    }
+
+    /**
+     * Set searchTerm.
+     *
+     * @param searchTerm as java.lang.String
```

Comment:
`java.lang.String` could be simplified to ` string` or the whole wording should be adjusted.